### DISCUSSION TITLE
Baremetall Install: missing directory

### DISCUSSION THREAD

**Discussion by ghost** [ellite/Wallos#552] [General]
If you download the code and unpack it to your own directory, there is missing at least on directory (potentially empty in the source): **images/uploads/logos**
So you are not able to add logos to subscriptions because imagick is unable to store files in this nonexisting directory. The resulting error message is cryptic and gives no hint for the root cause.
In the java console I also noticed to missing font-files: 
fa-regular-400.ttf and .woff2.

Otherwise a very good software. Congrats! I like it.

**Comment by ellite** [comment]
Hi. Thank you for the report. v2.27.3 should fix both issues.
Regards.

**Comment by ellite** [comment]
https://github.com/ellite/Wallos/issues/553
